lib/trader-sdk/lib/indicators/index.ts

```diff
@@ -0,0 +1,9 @@
+import * as BPercentMfiIndicator from './bb-percent-b-mfi';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/dills122/trader-tools/pullRequest?prid=8102010)
```suggestion
import * as BPercentMfiIndicator from "./bb-percent-b-mfi";
```